x-pack/legacy/plugins/lens/public/types.ts

```diff
@@ -139,8 +137,13 @@ export interface Datasource<T = unknown, P = unknown> {
   removeLayer: (state: T, layerId: string) => T;
   clearLayer: (state: T, layerId: string) => T;
   getLayers: (state: T) => string[];
+  removeColumn: (props: { prevState: T; layerId: string; columnId: string }) => T;
```

Comment:
Is there a specific reason, that we're using individual parameters for the other functions (like `removeLayer`, `clearLayer`), but now start using properties/options objects for this function?

I think it would be nicer if we don't mix those ways within the same interface, if there's no special reason for it.

Reply:
I didn't want to create a new name just for this, since the name is not used anywhere